apps/src/sites/studio/pages/maker/setup.js

```diff
@@ -111,15 +121,15 @@ const BoardSetupStatus = React.createClass({
      */
     function forEachLedInSequence(func, delay) {
       return new Promise(resolve => {
-        const leds = bc.prewiredComponents.colorLeds;
+        const leds = board.prewiredComponents_.colorLeds;
```

Comment:
Done.  After a painful day of learning how to test timed promise chains :sob:.  https://github.com/code-dot-org/code-dot-org/pull/13622